### PR TITLE
List Posts

### PR DESCRIPTION
**Pull request by BuiPhuongThaob**
## Related Tickets:
- [Task #27560](https://edu-redmine.sun-asterisk.vn/issues/27560)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by BuiPhuongThaob** [comment]
ready

**Comment by BuiPhuongThaob** [comment]
ready

File: app/Http/Controllers/HomeController.php [hunk 1]
```diff
@@ -3,11 +3,14 @@
...
 
 use Illuminate\Http\Request;
-
+use App\Models\Post;
 class HomeController extends Controller
 {
     public function index()
     {
-        return view('pages.home_page');
+        $posts = Post::latest()->limit(6)->get();
```
Comment (hoangminhuet) [inline app/Http/Controllers/HomeController.php]: đưa số vào config đi chị

**Review by huyenttm-0901 — COMMENTED** [review]
fix comment

**Comment by BuiPhuongThaob** [comment]
ready

**Pull request merged**